aea/aea.py

```diff
@@ -51,8 +51,8 @@ def __init__(
         resources: Resources,
         loop: Optional[AbstractEventLoop] = None,
         timeout: float = 0.0,
-        debug: bool = False,
-        programmatic: bool = True,
+        is_debug: bool = False,
```

Comment:
We now want all Boolean variables and functions with boolean return type to be prefixed with either `is_`, `has_` or `can_`.